src/components/Search/SearchAPI.js

```diff
@@ -0,0 +1,83 @@
+export const fetchAPIitems = async (searchParamsStr) => {
+  const searchURL = `${process.env.REACT_APP_STAC_API_URL}/search?${searchParamsStr}`
+
+  const response = await fetch(searchURL)
+  if (!response.ok) {
+    throw new Error(`An error has occurred: ${response.status}`)
+  }
+  const items = await response.json()
+  return items
+}
+
+export const fetchAggregatedItems = async (
+  searchParamsStr,
+  selectedCollection
+) => {
+  let preKey = ''
+  if (selectedCollection.startsWith('sentinel')) {
```

Comment:
```suggestion
  if (selectedCollection.startsWith('sentinel-2')) {
```
This gridding is only for sentinel-2-l1c and sentinel-2-l2a

Reply:
This has been updated.